pallets/lbp/src/lib.rs

```diff
@@ -504,7 +505,9 @@ pub mod pallet {
 
 				ensure!(pool.paused, Error::<T>::PoolIsNotPaused);
 
-				ensure!(Self::is_after_sale(&pool), Error::<T>::CannotUnpauseEndedPool);
+				// REVIEW: You want to ensure the pool is running, no? Or do you want to allow
```

Comment:
we want to allow paused start -> docs?

Reply:
sure, but the function did sth else AFAICT